crates/eips/src/eip4844/mod.rs

```diff
@@ -34,7 +34,7 @@ pub const BLS_MODULUS_BYTES: B256 =
 pub const BLS_MODULUS: U256 = U256::from_be_bytes(BLS_MODULUS_BYTES.0);
 
 /// Size a single field element in bytes.
-pub const FIELD_ELEMENT_BYTES: u64 = 32;
```

Comment:
It's breaking change, but I don't think it should break a lot of user-space code.

Reply:
can we keep this as u64 and avoid the breaking change, this would be more consistent with the other constant

Reply:
dbaa6efd0c87c035c39c1b38ea67025d36846197